articles/active-directory/saas-apps/ms-confluence-jira-plugin-adminguide.md

```diff
@@ -123,7 +123,29 @@ The following image shows the configuration screen in both Jira and Confluence:
 * **Enable Use of Application Proxy** checkbox, if you have configured your on-premise atlassian application in an App Proxy setup.
 
    * For App proxy setup , follow the steps on the [Azure AD App Proxy Documentation](../app-proxy/what-is-application-proxy.md).
-
+   * 
+## Release Notes
+
+|Plugin Version   |                                   Release Notes                                           |       Supported JIRA versions       |
+|-----------------|-------------------------------------------------------------------------------------------|-------------------------------------|
+|  1.0.20         |   Bug Fixes:                                                                              | Jira Core and Software:             |
+|                 |   1. JIRA SAML SSO add-on redirects to incorrect URL from mobile browser.                 |  7.0.0 to 9.5.0                     |
+|                 |   2. The mark log section after enabling the JIRA plugin.                                 |                                     |
+|                 |   3. JIRA plugin needs to report the last login date in the JIRA log.The issue is in Jira itself, the Last Login Date for a user does not update when sign in is via SSO.                                                                                      |                                     |
```

Comment:
minor sentence correction : the last login date for a user does not update **when user signs in via SSO**